plot_app/templates/index.html

```diff
@@ -16,42 +16,22 @@
 {{ info_table_html }}
 {{ error_labels_html }}
 
-{% if has_position_data %}
-<!-- Google Maps -->
-<div id="google-map"></div>
-<script>
+<div id="mapid"></div>
 
-var g_google_map = null;
-
-function initMap() {
-
-	var map = new google.maps.Map(document.getElementById('google-map'), {
-		zoom: 15,
-		streetViewControl: false,
-		zoomControl: true,
-		panControl: true,
-		mapTypeId: google.maps.MapTypeId.HYBRID,
-	});
-	g_google_map = map;
-
-	if(window.location.hostname == 'localhost') {
-		console.log('KML files do not work locally!'); //<-- important to know
-	}
-
-	var ctaLayer = new google.maps.KmlLayer({
-		// add some random parameter so that Google does not cache it (I had a
-		// case where the map did not work because Google cached it)
-		url: window.location.protocol+'//'+window.location.host+'/download'+
-			location.search+'&type=2&randnotused='+Math.random().toString(36).substring(3),
-		map: map
-	});
-}
-</script>
-<script async defer
-	 src="https://maps.googleapis.com/maps/api/js?key={{ google_maps_api_key }}&callback=initMap">
+<script>
+var lat_lon_focus = {{ lat_lon_focus }};
+var mymap = L.map('mapid').setView([lat_lon_focus.lat, lat_lon_focus.lon],17);
```

Comment:
I wish we could get the zoom automatic based on the area of log, I will try later, current we could use 15 as default

Reply:
I agree, something automatic would be better. You can use `fitBounds`, see here: https://gis.stackexchange.com/questions/76113/dynamically-set-zoom-level-based-on-a-bounding-box. It will also center it.